packages/charts/src/data-container/data-container.ts

```diff
@@ -203,15 +204,15 @@ export class DataContainer<DataPoint> {
     return this;
   }
 
-  getDataSeries(name: string | undefined): DataSeries<XYPoint> | undefined {
+  getDataSeries(name: string | undefined): DataSeries<XYPoint<any>> | undefined {
     if (!name) {
       return undefined;
     }
     this.processData();
     return this.series[this.namedData[name]];
   }
 
-  getAllDataSeries(): DataSeries<XYPoint>[] {
+  getAllDataSeries(): DataSeries<XYPoint<any>>[] {
```

Comment:
since currently we're only supporting `number` and `[number, number]` could we just type it explicitly like that instead of `any`? And then improve over that and try to make it generic?

Reply:
hmm, technically user can pass a `pointAccessor` function that will turn any input data ( that is being set with `setData` method ) into a `XYPoint`, we cannot enforce it to be just either `number` or `[number, number]`, but for the time being before generic implementation I think it would be fine :thinking:

Reply:
Ok, I've fiddled around and changing the type of both `getDataSeries` and `getAllDataSeries` methods, breaks most of views as they only operate on `y` being a `number` and I have to cast in various places which is not ideal :/